### PR TITLE
Updates for the release of v1.2.2

### PR DESCRIPTION
**Pull request by baigal628**
1) Added Change Log in README.md.
2) Updated Indexes genome for STAR was generated under STAR 2.7.6a but can be used for any STAR >= 2.7.4a.
genome.fa: ftp://ftp.ensembl.org/pub/release-99/fasta/homo_sapiens/dna/Homo_sapiens.GRCh(m)38.dna.primary_assembly.fa.gz
gtf: ftp://ftp.ensembl.org/pub/release-99/gtf/homo_sapiens/Homo_sapiens.GRCh(m)38.99.gtf.gz
3) Fixed typo in example documentation for scATAC-seq.
4) Added notice for LISA gene name requirements.

### CODE REVIEW

**Comment by crazyhottommy** [comment]
@taoliu the build failed, is it because taking too long?

**Comment by taoliu** [comment]
Let's approve it and merge the PR since it's all about documents.

File: example/RNA_infrastructure_10x/RNA_infrastructure_10x.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ Arguments  |  Description
...
+`--lisamode` | Mode to Run LISA, 'local' or 'web'. If the mode is set as 'local', please install [LISA](https://github.com/qinqian/lisa) and download pre-computed datasets following the instructions. The 'web' mode is to run online version of LISA. In consideration of the connection issue and size of datasets, the 'local' mode is recommended to run the whole MAESTRO pipeline. If the mode is 'local', please provide the name of LISA environment through `--lisaenv` and specify the directory where miniconda or anaconda is installed through `--condadir`. DEFAULT: local. (**Note:** LISA takes multiple types of gene sets which can be constituted of only official gene symbols, only RefSeq ids, only Ensembl ids, only Entrez ids, or a mixture of these identifiers.)
```
Comment (taoliu) [inline example/RNA_infrastructure_10x/RNA_infrastructure_10x.md:101]: Do we test both the local and web version of LISA in the version of MAESTRO?

File: README.md [hunk 2]
```diff
@@ -66,7 +74,7 @@ $ conda config --add channels conda-forge
 # To make the installation faster, we recommend using mamba
 $ conda install mamba -c conda-forge
 $ mamba create -n MAESTRO maestro=1.2.1.9999 -c liulab-dfci
```
Comment (taoliu) [inline README.md:76]: need to change it to 1.2.2

Comment (crazyhottommy) [reply README.md:76]: @baigal628 can you fix this and do another PR?

Comment (taoliu) [reply README.md:76]: I think @baigal628 can just push updates to her [baigal628:master](https://github.com/baigal628/MAESTRO). No need for another PR.

Comment (baigal628) [reply README.md:76]: Sure. What I previously did is that I commit to my fork baigal628:master and PR to this repo. I only tested the LISA local mode. To run in web mode, should we just download the DE gene list and upload it to the LISA website?

Comment (taoliu) [reply README.md:76]: @baigal628 I think the previous design is that the MAESTRO pipeline will 'connect to LISA website API' and retrieve the result. I don't know if it works now.

Sure. You can commit to your forked baigal628:master and we will see the updates in this PR as long as this PR is not closed.

**Comment by crazyhottommy** [comment]
I will merge it and @taoliu  you may release a 1.2.2 version.

**Pull request merged**